Rakefile

```diff
@@ -30,13 +30,6 @@ require 'fcrepo_wrapper'
 require 'active_fedora/rake_support'
 require 'engine_cart/rake_task'
 
-require 'jasmine'
```

Comment:
I am sorry that this consumed so much labor, this should have been removed at a much earlier date given these problems.